Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+certbot = "*"
+python-consul = "*"
+pyopenssl = "*"
```

Comment:
I intentionally did not pin the packets, since this way we can easily update package versions later.  At least for Certbot, we should update to the latest version every now and then, in case they fix issues with the ACME protocol or have general security fixes.  I've at least specified minimum versions now, which will allow easy updates but at least provide some more explicit information in the Pipfile.

Reply:
Sounds good, thanks for the min-pin!